src/library/scala/collection/parallel/mutable/ParArray.scala

```diff
@@ -179,7 +179,7 @@ self =>
 
     override def fold[U >: T](z: U)(op: (U, U) => U): U = foldLeft[U](z)(op)
 
-    override def aggregate[S](z: =>S)(seqop: (S, T) => S, combop: (S, S) => S): S = foldLeft[S](z)(seqop)
+    override def aggregate[B](z: =>B)(seqop: (B, T) => B, combop: (B, B) => B): B = foldLeft[B](z)(seqop)
```

Comment:
All the `par` type params were especially creatively non-conformist, as I recall from when I peeked under that hood/bonnet. It's too bad Dr Odersky didn't ask for straw man proposals for type param conventions for collections. Lets settle the fundamentals first, before building castles in the sky.